api/src/app/modules/forest-client/forest-client.controller.ts

```diff
@@ -22,14 +23,16 @@ export class ForestClientController extends BaseReadOnlyController<
   @ApiBearerAuth()
   @ApiResponse({ status: HttpStatus.OK, type: [ForestClientResponse], description: 'Returns only forest clients that the user is authorized for.' })
   async find(
-    @UserRequiredHeader() user: User,
+    @UserHeader() user: User,
   ): Promise<ForestClientResponse[]> {
     return this.service.find(user.clientIds);
   }
 
   @Get(':id')
+  @AuthGuardMeta(GUARD_OPTIONS.PUBLIC)
   @ApiResponse({ status: HttpStatus.OK, type: ForestClientResponse })
   async findOne(@Param('id', ParseIntPipe) id: number): Promise<ForestClientResponse> {
+    console.log("Forest Client findOne: ", id)
```

Comment:
Shouldn't use console.log (switch to logger or remove).

Reply:
Ha, was testing it and forgot to remove it.
Removed now.